autoshop/autoshop/middleware.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from analytics.models import AnalyticsRequestsRecord
+
+
+class AnalyticsMiddleware:
+
+    def __init__(self, response):
+        self.response = response
+
+    def __call__(self, request):
```

Comment:
Never try to access these private method. Please read docs related to `process_request` and `process_response`.